lib/components/narrative/narrative-itineraries.js

```diff
@@ -556,8 +536,24 @@ const reduceErrorsFromResponse = (acc, cur) => {
   return acc
 }
 
-// connect to the redux store
+/**
+ * If there are no itineraries, generate an error
+ * so we're not left with an empty results container.
+ */
+const createNoItinerariesErr = (state, errors, pending) => {
+  const itineraries = getActiveItineraries(state)
+  const search = getActiveSearch(state)
+  if (errors && itineraries.length === 0 && !pending) {
+    errors.NO_TRANSIT_CONNECTION = new Set()
+  }
+  if (search?.response?.some((res) => res.error)) {
+    errors.SYSTEM_ERROR = new Set()
+  }
 
+  return errors
```

Comment:
We have put in a ton of effort to explicitly avoid doing this. The errors should be generated within the datastore, not in the view handler